lib/multiply.rb

```diff
@@ -1,2 +1,15 @@
 def multiply(num1, num2)
+    
+    i = 0
+    answer = 0
+    
+    while(i<num1)
+    
+    answer = answer + num2
```

Comment:
a_var += another_var

I've used it in previous assignments, but for this one I remember using the phrase "The answer needs to be itself plus the second number" and I just typed it up as that.